code/modules/projectiles/guns/specialist.dm

```diff
@@ -127,7 +127,7 @@
 	. = ..()
 
 /obj/item/weapon/gun/rifle/sniper/M42A/process()
-	if(!zoom)
+	if(!rail.zoom)
```

Comment:
i guess the m42A rail scope cant be removed ? would suck to have this runtime

Reply:
no